messages/gl/LC_MESSAGES/comix.po

```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "POT-Creation-Date: \n"
 "PO-Revision-Date: \n"
-"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
+"Last-Translator: Marcelo G�es <vanquirius@gentoo.org>\n"
```

Comment:
what is this change ? Did you change the file's encoding somehow ?
Github/my browser are not rendering it properly anymore ...